src/lib/response-body-listener.ts

```diff
@@ -20,17 +20,22 @@ export class ResponseBodyListener {
       details.requestId,
     ) as any;
 
-    const decoder = new TextDecoder("utf-8");
+    // const decoder = new TextDecoder("utf-8");
     // const encoder = new TextEncoder();
 
-    let responseBody = "";
+    let responseBody = new Uint8Array();
     filter.ondata = event => {
```

Comment:
This doesn't actually save the content hash. I've verified that the row in the dataset is empty when running the tests locally. This isn't caught by the test suite because we disabled the test (see: https://github.com/mozilla/OpenWPM/blob/9d0b99cacef95235685d9bef062f667b04e6e0fb/test/test_http_instrumentation.py#L614-L615).

Mind to re-enable this test in https://github.com/mozilla/OpenWPM/pull/309 and get it saving correctly here?

Reply:
Content hash saving has been implemented in f9f912d38e1de5adf19bf7130b4cc60f8b545dbc and the corresponding test can be found here: https://github.com/eriktews/OpenWPM/commit/29b127c97e261e4988babaf553c54abc50ea9bf0 and it seems to work fine now.

Reply:
Thanks!